src/index.html

```diff
@@ -8,46 +8,52 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <div class="wrapper">
-      <div class="card wrapper__card" data-qa="card">
-
-        <div class="card__image card-top">
-          <img
-            src="./images/imac.jpeg"
-            class="img"
-            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
-          >
-        </div>
-
-        <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
-        <div class="card__code">Product code: 195434</div>
-        <div class="card__rate">
-
-          <ul class="stars stars--4">
-            <li class="stars__star"></li>
-            <li class="stars__star"></li>
-            <li class="stars__star"></li>
-            <li class="stars__star"></li>
-            <li class="stars__star"></li>
-          </ul>
-
-          <div class="card__reviews">Reviews: 5</div>
-
-        </div>
-
-        <div class="card__price">
-          <span class="text">Price:</span>
-          <span class="value">$2,199</span>
-        </div>
-
-        <a href="#"
-           class="card__link"
-           data-qa="hover">
-           Buy
-        </a>
+
+    <div class="card" data-qa="card">
+
+      <div class="card__image card-top">
+
+        <img
+          src="./images/imac.jpeg"
+          class="img"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+      </div>
+
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__rate">
+
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <div class="card__reviews">Reviews: 5</div>
```

Comment:
There shouldn't be any spaces between parent and child structures, like between this div and the ul.

Reply:
➕ , fix all caes